packages/@uppy/companion/package.json

```diff
@@ -41,7 +41,7 @@
     "express-interceptor": "^1.2.0",
     "express-prom-bundle": "^3.1.0",
     "express-session": "1.15.6",
-    "grant-express": "^4.0.1",
+    "grant-express": "4.1.2",
     "helmet": "3.8.2",
```

Comment:
Why this version? And should we pin everything else too?

Reply:
that's  the last version installed in package-lock.json